manager/container.go

```diff
@@ -306,8 +322,8 @@ func newContainerData(containerName string, memoryCache *memory.InMemoryCache, h
 	cont := &containerData{
 		handler:                  handler,
 		memoryCache:              memoryCache,
-		housekeepingInterval:     *HousekeepingInterval,
-		maxHousekeepingInterval:  maxHousekeepingInterval,
+		housekeepingInterval:     jitter(*HousekeepingInterval, JitterFactor),
```

Comment:
Agreed. I assume the point of this is to spread the collection across the housekeeping interval? I don't think this should affect the actual housekeeping interval but instead just add a random sleep (e.g. with `timer.After`) in to the housekeeping loop?

The aim must be to collect from all containers in the effective period between housekeeping runs?